libbeat/licenser/elastic_fetcher.go

```diff
@@ -98,6 +98,7 @@ func (f *ElasticFetcher) parseJSON(b []byte) (License, error) {
 
 // esClientMux is taking care of round robin request over an array of elasticsearch client, note that
 // calling request is not threadsafe.
+// nolint: unused // it's used on Linux
```

Comment:
```suggestion
//nolint:unused // it's used on Linux
```

are u sure this comment is working?